test/test/mocks/mfe.js

```diff
@@ -1,65 +1,78 @@
 export const LIST_MFE_RESPONSE_OK = [
   {
-    id: 'app-builder-header',
-    target: 'primary-header',
+    id: 'layout-core-bundle:app-builder-header',
     name: 'app-builder-header',
-    assets: ['mocks/app-builder-header/main.js', 'mocks/app-builder-header/main.css'],
+    slot: 'primary-header',
+    assets: ['main.js', 'main.css'],
     customElement: 'app-builder-header',
-    params: {
-      api: {
-        headerMS: 'ms/app-builder-header-ms',
+    api: [
+      {
+        name: 'ms1',
+        baseUrl: 'http://localhost:8080',
       },
-    },
+      {
+        name: 'ms2',
+        baseUrl: 'http://localhost:8090',
+      },
+    ],
   },
   {
-    id: 'app-builder-menu',
-    target: 'primary-menu',
+    id: 'layout-core-bundle:app-builder-menu',
     name: 'app-builder-menu',
-    assets: ['mocks/app-builder-menu/main.js', 'mocks/app-builder-menu/main.css'],
+    slot: 'primary-menu',
+    assets: ['main.js', 'main.css'],
     customElement: 'app-builder-menu',
-    params: {
-      api: {
-        headerMS: 'ms/app-builder-header-ms',
+    api: [
+      {
+        name: 'ms1',
+        baseUrl: 'http://localhost:8080',
+      },
+      {
+        name: 'ms2',
+        baseUrl: 'http://localhost:8090',
       },
-    },
+    ],
   },
   {
-    id: 'example-mfe',
-    target: 'content',
-    name: 'content',
-    assets: ['mocks/example-mfe/main.js', 'mocks/example-mfe/main.css'],
-    customElement: 'example-mfe',
-    params: {
-      api: {
-        headerMS: 'ms/app-builder-header-ms',
+    id: 'layout-core-bundle:strapi-content-template',
+    name: 'strapi-content-template',
+    slot: 'content',
+    assets: ['main.js', 'main.css'],
+    customElement: 'app-builder-menu',
+    paths: ['/'],
+    api: [
+      {
+        name: 'ms1',
+        baseUrl: 'http://localhost:8080',
+      },
+      {
+        name: 'ms2',
+        baseUrl: 'http://localhost:8090',
       },
-    },
-    activeRoutes: ['/example-mfe'],
+    ],
```

Comment:
for this one, we need to keep the data related to `example-mfe` (it's the local mock)  but with the new structure